docs/topics/mta-import-questionnaire.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+
+
+:_content-type: PROCEDURE
+[id="mta-import-questionnaire_{context}"]
+= Import questionnaire
+
+.Procedure
+
+. To import a questionnaire, click *Import questionnaire*.
+. Click *Upload*.
+. Navigate to the location of your questionnaire, and click *Open*.
+. Click *Import*.
+
+[WARNING]
+.UNIQUE constraint failed: Questionnaire.Name
+====
```

Comment:
Formatting of ".UNIQUE..."  Should be  ` blah blah ` and probably should be inside = signs.

Reply:
https://docs.asciidoctor.org/asciidoc/latest/blocks/admonitions/